ramls/check-in-by-barcode-request.json

```diff
@@ -16,6 +16,14 @@
       "description": "When the item was checked in",
       "type": "string",
       "format": "date-time"
+    },
+    "claimedReturnedResolvedBy": {
+      "description": "Defines who has resolved claimed returned item state",
+      "type": "string",
+      "enum": [
+        "Found by library",
+        "Returned by patron"
+      ]
```

Comment:
I think we should proceed with the kind of resolution, should I rename it to: `claimedReturnedResolutionReason`? Any other suggestions?

Reply:
> I think we should proceed with the kind of resolution, should I rename it to: `claimedReturnedResolutionReason`? Any other suggestions?

My suggestion would be `claimedReturnedResolution`. 

In conversations with Emma, how does she refer to this, does she call it a resolution or a reason or something else? It might be worth asking her.

Reply:
Renamed to `claimedReturnedResolution`